### PR TITLE
Closes #129. Clarifying the use of wallet string address

### LINKED ISSUE #129: Possible typo and clarification edits on README.md
Hi all. I would like to propose the following edits to README.md. They are minor in nature, but should be helpful to users first getting in contact with this repo.

* In the *Run the code* section, I believe there should be a hyphen between "docker" and "compose", like so:

Run the System
```
# docker-compose --file docker-compose-2pc.yml up --build
```

* In the *Setup test wallets and test them* section, I would propose adding a one-row explainer that the references to wallet1 are the result of the previous step. Users replicating this code locally might not be aware of that and think that their code is wrong for some reason, when in fact it's just a matter of substituting the string in the instructions for whatever string was the output in their own local version.

### PR DESCRIPTION
**Pull request by dkgaraujo**
Signed-off-by: Douglas Araujo <dkg.araujo@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,7 +150,9 @@ In each of the below commands, you should pass `atomizer-compose.cfg` instead of
   usd1qrw038lx5n4wxx3yvuwdndpr7gnm347d6pn37uywgudzq90w7fsuk52kd5u
   ```
 
-* Send currency from one wallet to another (e.g., 30 atomic units of currency)
+  Note the resulting string ("usd1qrw038lx5n4wxx3yvuwdndpr7gnm347d6pn37uywgudzq90w7fsuk52kd5u"). It is the new wallet's address, and will be used in the following step.
```
Comment (HalosGhost) [inline README.md]:
```suggestion
  Note the resulting string ("usd1qrw038lx5n4wxx3yvuwdndpr7gnm347d6pn37uywgudzq90w7fsuk52kd5u").
  It is the new wallet's address, and will be used in the following step.
```

File: README.md [hunk 2]
```diff
@@ -150,7 +150,9 @@ In each of the below commands, you should pass `atomizer-compose.cfg` instead of
...
+
+* Send currency from one wallet to another (e.g., 30 atomic units of currency) **Note that you may need to change the wallet string in the code below ("usd1qrw0...") depending on the result of the previous step.**
```
Comment (HalosGhost) [inline README.md]:
```suggestion
* Send currency from one wallet to another (e.g., 30 atomic units of currency).
  **Note that you will need to change the wallet string in the code below ("usd1qrw0...") depending on the result of the previous step.**
```

File: README.md [hunk 3]
```diff
@@ -171,7 +173,7 @@ In each of the below commands, you should pass `atomizer-compose.cfg` instead of
   Balance: $0.20, UTXOs: 4, pending TXs: 0
   ```
 
-* Import coins to the receiving wallet
+* Import coins to the receiving wallet. Note that the string after `importinput` is the same as the result from step above of sending the currency between wallets.
```
Comment (HalosGhost) [inline README.md]:
```suggestion
* Import coins to the receiving wallet.
  Note that the string after `importinput` is the same as the result from step above of sending the currency between wallets.
```

**Review by HalosGhost — CHANGES_REQUESTED** [review]
I think this kind of clarity is good, but I do have a couple of requests.

I made a few suggestions inline to keep one-sentence-per-line (a gift for reviewers of diffs); but I also am a bit worried that some of the specificity might actually be a bit too rigid.

This walkthrough is meant to get people interacting with the system quickly, but this is not a production system, doesn't follow semver, and the CLI (what little of it there is) is purely for testing and demonstration. To that end, I wonder if it might make more sense (from a documentation-burden standpoint) to have more general statements (e.g., “send currency from the first wallet to the address of the second wallet created in the previous step”) which are more likely to remain evergreen.

Thoughts?

File: README.md [hunk 4]
```diff
@@ -174,7 +171,7 @@ In each of the below commands, you should pass `atomizer-compose.cfg` instead of
   Balance: $0.20, UTXOs: 4, pending TXs: 0
   ```
 
-* Import coins to the receiving wallet. Note that the string after `importinput` is the same as the result from step above of sending the currency between wallets.
+* Import coins to the receiving wallet using the string after `importinput` from step above.
```
Comment (dkgaraujo) [inline README.md:174]: I tried to follow a similar one-line style for this string as well.

**Review by dkgaraujo — COMMENTED** [review]
Thank you for the suggestion to further simplify. It is much better indeed. I hope this new version can address that.

**Comment by dkgaraujo** [comment]
FYI, tThe recent commits are actually a rebase because one of them was not signed off. The content is the same.

**Comment by dkgaraujo** [comment]
Hi @HalosGhost, when you have some time, please feel free to let me know if there is anything outstanding from my side in this PR. I can't seem to find anything else outstanding after I pushed the simpler version.

**Comment by HalosGhost** [comment]
@dkgaraujo I'm sorry about the delay! My last week or two has been quite full. I'm making time to go through all the open PRs in the next few days!

**Comment by dkgaraujo** [comment]
> @dkgaraujo I'm sorry about the delay! My last week or two has been quite full. I'm making time to go through all the open PRs in the next few days!

No issue - didn't mean to press, I was just checking in case I forgot anything. Happy to proceed at your pace.

**Review by HalosGhost — APPROVED** [review]
Simple clarity improvement to our README walkthrough.

Looks great!

**Comment by HalosGhost** [comment]
@dkgaraujo Whenever you're ready, if you can squash these to a single commit, I'll happily merge!

**Comment by HalosGhost** [comment]
@dkgaraujo it looks like you merged in from `trunk`, but didn't rebase/squash. The easiest way to get yourself back to a good spot is probably (assuming you have this fork configured as a remote named `upstream` on your clone) `git pull --rebase upstream trunk`; after that point, if you still need to squash, you can do an interactive rebase.

**Comment by HalosGhost** [comment]
@dkgaraujo let me know if you need help squashing.

**Comment by dkgaraujo** [comment]
> @dkgaraujo let me know if you need help squashing.

@HalosGhost sorry for the delay - I had to prioritise another project. I'll come back to this by tomorrow at the latest. Thanks for the patience.

**Comment by HalosGhost** [comment]
> sorry for the delay

No worries; I'll still be here to review when you're ready!

**Comment by dkgaraujo** [comment]
Hi @HalosGhost, thanks again for the patience. I successfully rebased and squashed (I believe) - but in the process of interactively rebasing I noted that I went a couple of steps back from my own first commit, and when I removed the previous commits by other contributors from the interactive page, it effectively deleted them from my version of trunk. So please don't merge it. I think going back and making sure all is there would be an additional complication on this already complicated PR. Maybe we can just cancel this one and reopen another one with a simple, single commit implementing the changes? Happy to hear your views.

**Pull request closed without merging**